kubeflow/core/tensorboard.libsonnet

```diff
@@ -0,0 +1,327 @@
+{
+  // Parameters are intended to be late bound.
+  params:: {
+    name: null,
+    numGpus: 0,
```

Comment:
A lot of these parameters don't look relevant for tensorboard. e.g.
we don't need numGPUs
modelName
modelPath
httpProxyImage

We do need parameters for
tbImage - Image for tensorboard to use
logDir - the directory where TensorFlow logfiles are stored (containing the data for TB).

Reply:
I changed the parameters. I have two questions:

1- Should I initialize the logDir parameter as an empty string since it will be something that will be typed in or is there a better way to do it?
 ```
logdir = ""
```
2- I used the `gcr.io/kubeflow-images-staging/tensorflow-cpu:1.6.0-latest-agwl` from `gcr.io/kubeflow-images-staging` but is this the right one to use?

```
 params:: {
    name: null,
    labels: {
      app: $.params.name,
    },
    serviceType: "ClusterIP",
    logDir: "",
    defaultTbImage: "gcr.io/kubeflow-images-staging/tensorflow-cpu:1.6.0-latest-agwl",

    // Whether or not to enable s3 parameters
    s3Enable:: false,

    // Which cloud to use
    cloud:: null,
  },
```

Thanks!

Reply:
1. I think iniitalizing to the empty string and using an std.assertEqual to ensure its set is the best option
http://jsonnet.org/docs/stdlib.html

2. The default image should be the most recent stable image of TensorFlow.
https://pantheon.corp.google.com/gcr/images/tensorflow/GLOBAL/tensorflow?gcrImageListsize=50

So probably
gcr.io/tensorflow/tensorflow:latest

Reply:
So in the `tensorboard.jsonnet` will it be something like this:
```
local logDir = import "param://logDir";
std.assertEqual(logDir, "/tmp/tensorflowlogs")
```